pkg/sso/sso.go

```diff
@@ -16,9 +16,11 @@ var (
 // Configure configures the default client that is used by GetAuthorizationURL,
 // GetProfile and Login.
 // It must be called before using those functions.
-func Configure(apiKey, projectID string) {
+// Deprecated: Please pass clientID as an argument, not projectID.
+func Configure(apiKey, projectID string, clientID string) {
```

Comment:
Should we just rename `projectID` to `clientID` here? Since these aren't explicitly used by developers, we can get away with a replace.